packages/graphql-typescript-definitions/test/__snapshots__/cli.test.ts.snap

```diff
@@ -70,17 +70,18 @@ Object {
     "signal": null,
   },
   "stderr": "",
-  "stdout": "
- ERROR  Error parsing 'packages/graphql-typescript-definitions/test/fixtures/missing-schema/schema.json':
-
-ENOENT: no such file or directory, open 'packages/graphql-typescript-definitions/test/fixtures/missing-schema/schema.json'
-Error: Error parsing 'packages/graphql-typescript-definitions/test/fixtures/missing-schema/schema.json':
-
-ENOENT: no such file or directory, open 'packages/graphql-typescript-definitions/test/fixtures/missing-schema/schema.json'
+  "stdout": " ERROR  schemaPath not found for project 'GraphQL': 'packages/graphql-typescript-definitions/test/fixtures/missing-schema/schema.json'
```

Comment:
Maybe instead of just printing the path, say “expected to find it in X based on GraphQL config file, but no such file existed.”